### PR TITLE
travis: add a small encode/decode smoke test

### PR DESCRIPTION
**Pull request by kylophone**

### CODE REVIEW

**Comment by 1480c1** [comment]
I think that libaom may need to be build from source because it does not seem to be available for xenial

File: .travis.yml [hunk 1]
```diff
@@ -15,13 +15,12 @@ jobs:
    # General Linux build job
    - name: Build
      before_install:
-     - sudo apt-get install -y libaom-dev valgrind
+     - sudo apt-get install -y valgrind
      script:
      - cd Build/linux
      - ./build.sh release
      - wget -nc https://mf4.xiph.org/~ltrudeau/videos/nyan.y4m
      - valgrind --leak-check=full ../../Bin/Release/SvtAv1EncApp -i nyan.y4m -enc-mode 0 -lp 1 -q 24 -o out.yuv -b out.ivf
```
Comment (hassount) [inline .travis.yml]: + this looks good, it might be a little too slow though, could we maybe run -enc-mode 4 or 5 ?

**Comment by 1480c1** [comment]
Maybe it would be better to enable the auto stop builds if there is an overriding commit for travis so that way there isn't a backlog of builds.
image

**Comment by 1480c1** [comment]
@kylophone You may need to add `--no-check-certificate` to the wget command since it seems that wget thinks that xiph's cert is invalid

File: .travis.yml (hunk 1 above)
Comment (joelsole) [reply .travis.yml]: Fewer frames should be enough.  Somewhere from -n 2  to  -n 8.

**Pull request merged**